src/frontend/src/app/models/mock-events.ts

```diff
@@ -0,0 +1,15 @@
+import { Event } from '../models/event';
+
+export const EVENTS: Event[] = [
+  { name: 'Camp 1', date: '08/21/2019', event_type: 'K12/Retreat/Field Trip', organizer: 'Name and/or phone number' },
```

Comment:
Not sure if we want to include mock data in the frontend as apposed to using the Django API. That way we can adjust data on the backend without adjusting the frontend.

Reply:
Mock data should get deleted when #58 gets completed

Reply:
Gotcha, that makes sense. I'll add some more specific description details so we know that story should include API data.